src/torchcodec/_core/custom_ops.cpp

```diff
@@ -394,8 +394,6 @@ void encode_audio_to_file(
       .encode();
 }
 
-// TODO-ENCODING is "format" a good parameter name?? It kinda conflicts with
-// "sample_format" which we may eventually want to expose.
```

Comment:
torchaudio.load() uses `format` as well, so I think that's fine. We haven't discussed the public API yet anyway.